### PR TITLE
Sort active people by "most active" to "least active" when running ":stats"

### PR DESCRIPTION
**Pull request by xabolcs**
Fix for #73.

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -450,7 +450,9 @@ Stats.prototype.generateStats = function(metrcs, from) {
       report = report + keys[i] + ":  " + metrcs[keys[i]].join(", ") + "\n";
     } else {
       if (keys[i] == "activeUsers") {
-        var speakers = Object.keys(metrcs.activeUsers);
+        var speakers = Object.keys(metrcs.activeUsers).sort(function (a, b) {
```
Comment (xabolcs) [inline bot.js]: node.js doesn't support fat arrow functions yet.

**Comment by xabolcs** [comment]
This is ready to review!

**Comment by whimboo** [comment]
That looks fine to me. Would you mind showing me an excerpt of the output for an example testday?

**Comment by xabolcs** [comment]
With the following `metrics.json`
``` json
{"topic":"topic","etherpad":"etherpad","start":"2015-12-13T23:30:00.000Z","end":"2015-12-13T23:45:00.000Z", "activeUsers": { "fistSpeaker": 5, "secondSpeaker": 6, "thirdSpeaker": 8, "fourthSpeaker": 7, "lastSpeaker": 9 },"hourUTC":{},"firebotBugs":[],"optOutTotal":0}
```
I had the following conversation with my bot:
```
[21:02:49]	xabolcs	:stats
[21:02:49]	xabolcs_tdb	topic: topic
[21:02:49]	xabolcs_tdb	etherpad: etherpad
[21:02:49]	xabolcs_tdb	start: Mon Dec 14 2015 00:30:00 GMT+0100 (CET)
[21:02:49]	xabolcs_tdb	end: Mon Dec 14 2015 00:45:00 GMT+0100 (CET)
[21:02:49]	xabolcs_tdb	The following 5 people were active in the channel: lastSpeaker: 9, thirdSpeaker: 8, fourthSpeaker: 7, secondSpeaker: 6, fistSpeaker: 5
[21:02:49]	xabolcs_tdb	The following hours (UTC) were active in the channel: firebotBugs:
[21:02:49]	xabolcs_tdb	optOutTotal: 0
```

**Comment by whimboo** [comment]
Great! I got this PR merged after updating the commit message to our standard format as https://github.com/mozilla/testdaybot/commit/581a8ffbd6bc618b1d31471e991e394e115ffe22

**Pull request closed without merging**